Vejleder comments (#53)
MrLongbottom/P10-Article: sections/plate_notation.tex

For these documents, one of the taxonomies is chosen randomly for each word in the document.

\subsection{Combinations}
\vejleder[inline]{hvorfor og formål?}

> We have created a combination model to see whether using multiple meta-data fields at the same time to draw topics, would have an effect on the performance of the topic model.

Reviewer: ```suggestion
We have created a combination model to see whether using multiple metadata fields at the same time to draw topics, would affect the performance of the topic model.
```

> The first model, we have created is the Author-Category combination model.

Reviewer: ```suggestion
The first model we have created is the Author-Category combination model.
```

> To combine the Author and Category metadata, we simply multiply the topic distributions $\theta_a$ and $\theta_c$, when drawing a new topic.
> Then, when sampling for a new word topic, we normalize the product of the multiplication to sum to 1, which also makes it a distribution. \vejleder[inline]{mathify this}

Reviewer: ```suggestion
To combine the Author and Category metadata, we multiply the topic distributions $\theta_a$ and $\theta_c$ when sampling a new word topic.
Though, before sampling the word topic, we normalize the product of the multiplication to sum to 1, which also makes it a distribution. \vejleder[inline]{mathify this}
```
